connectors/spark/src/main/scala/io/unitycatalog/connectors/spark/UCSingleCatalog.scala

```diff
@@ -171,6 +171,14 @@ private class UCProxy extends TableCatalog with SupportsNamespaces {
         GcsVendedTokenProvider.ACCESS_TOKEN_KEY -> gcsCredentials.getOauthToken,
         GcsVendedTokenProvider.ACCESS_TOKEN_EXPIRATION_KEY -> temporaryCredentials.getExpirationTime.toString
       )
+    } else if (uri.getScheme == "abfs" || uri.getScheme == "abfss") {
+      val azCredentials = temporaryCredentials.getAzureUserDelegationSas
+      Map(
+        "fs.azure.account.auth.type" -> "SAS",
+        "fs.azure.account.hns.enabled" -> "true",
+        "fs.azure.sas.token.provider.type" -> "io.unitycatalog.connectors.spark.AbfsVendedTokenProvider",
+        "fs.azure.sas.fixed.token" -> azCredentials.getSasToken,
```

Comment:
shall we put these string constants in `AbfsVendedTokenProvider`?

Reply:
Good call!  I cleaned that up, and pulled in the constants from abfs in cases where we were using already established conf values.